sql/mob_droplist.sql

```diff
@@ -1006,17 +1006,19 @@ INSERT INTO `mob_droplist` VALUES (175,0,0,1000,3179,100);
 INSERT INTO `mob_droplist` VALUES (175,0,0,1000,3180,100);
 INSERT INTO `mob_droplist` VALUES (175,0,0,1000,3185,100);
 INSERT INTO `mob_droplist` VALUES (175,0,0,1000,18967,440);
-INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1449,0); -- (Goblin lottery NMs, Be)
+INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1449,0); -- (Goblin lottery NMs, Dynamis Be) tukuku_whiteshell
 INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1449,90);
-INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1450,10);
-INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1452,0);
+INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1450,10); -- lungo-nango_jadeshell
+INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1452,0); -- ordelle_bronzepiece
 INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1452,10);
-INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1453,10);
-INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1455,0);
+INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1453,10); -- montiont_silverpiece
+INSERT INTO `mob_droplist` VALUES (176,2,0,1000,1455,0); -- one_byne_bill
 INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1455,90);
-INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1456,10);
-INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1470,80);
-INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1520,80);
+INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1456,10); -- one_hundred_byne_bill
+INSERT INTO `mob_droplist` VALUES (176,0,0,1000,1470,80); -- sparkling_stone
```

Comment:
either 10% or 5%. ffxi db can be used to determine witch one, based on th2 information they have (this applies to all the ones im going to point out)

Reply:
I'm going with 5% since any nm with good amount of data usually show below 5%.